src-tauri/src/config.rs

```diff
@@ -197,6 +197,7 @@ pub struct LauncherConfig {
   pub locale: Option<String>,
   pub mod_sources: Option<Vec<String>>,
   pub decompiler_settings: Option<DecompilerSettings>,
+  pub autoupdate: bool
```

Comment:
this should be an `Option<bool>` so we can load JSON files that don't yet have this option yet -- otherwise the parsing will fail.

I'd also recommend changing the name to something that reflects the purpose better.  `check_for_latest_mod_version` perhaps?